Changelog.md

```diff
@@ -15,6 +15,7 @@
 ## [v2.3.3]
 - Fix bug where uploading scanned exam pages with overwriting option selected did not update submission files (#6768)
 - Fix bug: "Download Submissions" download link was not being rendered from partial view (#6779)
+- Allow inactive groups in the graders tab to be displayed (#6778)
```

Comment:
This entry should go under the "unreleased" section above.

Also, please reword to "Allow inactive groups in the graders table to be toggled for display". (Since they were originally already displayed, this change makes it so that you can _hide_ these groups.)